lab-cameron/test/fs-readfile-test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const expect = require('chai').expect;
+const reader = require('../lib/fs-readfile.js');
+
+let filePaths = [
+  `${__dirname}/../data/first.txt`,
+  `${__dirname}/../data/second.txt`,
+  `${__dirname}/../data/third.txt`,
+];
+
+let hexBuffers = reader(filePaths);
```

Comment:
**TODO FOR LOST POINTS**

All your tests are not passing because you are getting undefined values. That is due to the asynchronous nature of the the ``reader`` function. You can't assign a an async method to a variable because you never know when you'll receive your outputted data
In order to test properly that your hexBuffer is the proper output, you will have to put your ``expect`` statements in the context of your callback functions. After the modification I've made to your ``reader`` module, here's a sample test I've written for you:

```
let testData = [ '6669727374207465', '7365636f6e642074', '7468697264207465' ]

describe('fs-readfile.js', function() {

  describe('judy testing', function(){
    it('should return some hex', function(done) {
      //since I've modified your reader function to take in a callback, I am invoking it accordingly here. Your expect statements go in the body of that callback function so you can make comparisons with your returned buffer and the testData array I made above.
      reader(filePaths, function(hexBuffers){
        console.log(hexBuffers, 'what is here?');
        expect(typeof hexBuffers[0]).equal(typeof 'string');
        expect(hexBuffers[0]).to.equal(testData[0]);
        expect(hexBuffers).to.deep.equal(testData);
        done();
      });
    });
  });
```
I hope this helps push you in the right direction if you want to resubmit. See me if you need help!